pages/pricing/_other-services.html

```diff
@@ -13,5 +13,12 @@ <h2>Other Services</h2>
         </tr>
       </tbody>
     </table>
+
+    <h3>Miscellaneous Fees</h3>
+    <p><strong>Gruntwork has no hidden fees.</strong> The one fee we apply is a one-time charge of $4,000 for any significant requested
+    changes on our contract to cover the cost we pay to our lawyers to review this. Minor contract change requests are
+    done inhouse and cost nothing.</p>
```

Comment:
Although I like the sentiment, and we really don't hide any fees, we also don't display our enterprise pricing or consulting pricing on the website. Perhaps just keep it simple and say "Legal fee": the pricing on the page comes with our standard contract. If you request *significant* changes on that contract, we add a fee of XXX to pay our lawyers. Minor contract changes...